src/Windows/Avalonia.Direct2D1/ExternalRenderTarget.cs

```diff
@@ -43,5 +44,13 @@ public IDrawingContextImpl CreateDrawingContext(IVisualBrushRenderer visualBrush
                 }
             });
         }
+
+        public IRenderTargetBitmapImpl CreateLayer(int pixelWidth, int pixelHeight)
+        {
+            var platform = AvaloniaLocator.Current.GetService<IPlatformRenderInterface>();
+
+            // TODO: Get proper DPI here.
+            return platform.CreateRenderTargetBitmap(pixelWidth, pixelHeight, 96, 96);
```

Comment:
Why are you calling `platform.CreateRenderTargetBitmap` here? It will just create a WIC render target.

Reply:
Ah yeah, will fix.